frontend/fleet/helpers.ts

```diff
@@ -646,7 +646,7 @@ export const secondsToDhms = (d: number): string => {
   return dDisplay + hDisplay + mDisplay + sDisplay;
 };
 
-export const syntaxHighlight = (json: JSON): string => {
+export const syntaxHighlight = (json: any): string => {
```

Comment:
Is there a reason for changing this to any?

Reply:
Oh, this was a mistake. I was playing with putting non-JSON in there because the mocks had something like `... more responses` in the JSON! Turns out our syntax highlighter can't handle non-JSON, so I just took out the `... more responses` but forgot to turn this back to type JSON